app/models/bgs_dependents/marriage.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module BGSDependents
+  class Marriage < Base
+    attribute :ssn, String
+    attribute :first, String
+    attribute :middle, String
+    attribute :last, String
+    attribute :suffix, String
+    attribute :vet_ind, String
+    attribute :birth_date, String
+    attribute :alt_address, String
+    attribute :va_file_number, String
+    attribute :lives_with_vet, String
+    attribute :ever_married_ind, String
+    attribute :martl_status_type_cd, String
+
+    def initialize(dependents_application)
+      @dependents_application = dependents_application
+      @spouse_information = @dependents_application['spouse_information']
+
+      self.attributes = spouse_attributes
+    end
+
+    def format_info
+      attributes.with_indifferent_access
+    end
+
+    def alt_address
+      @dependents_application.dig('does_live_with_spouse', 'address')
+    end
+
+    def address(marriage_info)
+      dependent_address(
+        @dependents_application,
+        marriage_info['lives_with_vet'],
+        marriage_info['alt_address']
+      )
+    end
```

Comment:
It even seems like `:address` could be an attribute of this model 🤷

Reply:
> Why does this method accept a marriage_info input param
You are correct, thank you for pointing that out. It's left-over from a refactor.

> I'm also not understanding why it needs to be defined in base.rb. Looks like it could just be written as a one-liner
We have the `dependent_address ` method in base because it's used by another class as well (other class is in another PR). If I try the one-line approach, it goes beyond our line length constraint.

> It even seems like :address could be an attribute of this model
Yeah, that is a great idea, silly I didn't think of it 🤦